account_manager/src/validator/deposit.rs

```diff
@@ -91,7 +103,10 @@ pub fn cli_run<T: EthSpec>(
         PathBuf::new().join(".lighthouse").join("validators"),
     )?;
     let validator: String = clap_utils::parse_required(matches, VALIDATOR_FLAG)?;
-    let eth1_ipc_path: PathBuf = clap_utils::parse_required(matches, ETH1_IPC_FLAG)?;
+    let eth1_ipc_path: PathBuf =
+        clap_utils::parse_optional(matches, ETH1_IPC_FLAG)?.unwrap_or_else(|| PathBuf::new());
```

Comment:
I think it would be idiomatic to keep these as an `Option` instead of unwrapping to a default value. `Option` is a great way to avoid the ambiguity of using a default value to represent an uninitialized value.

Here's an example of use dealing with two optional and conflicting flags:

https://github.com/sigp/lighthouse/blob/7d897a05196ac2f06171df899dcd339ed9677c03/account_manager/src/validator/create.rs#L135-L146

Reply:
thanks for the hint! done!